test/e2e/mnist_raycluster_sdk.py

```diff
@@ -18,11 +18,14 @@
     min_memory=0.5,
     max_memory=1,
     num_gpus=0,
+    image="quay.io/project-codeflare/ray:2.5.0-py38-cu116",
```

Comment:
Why do you use custom image in here?

Reply:
I took same reference from the demo notebooks examples(https://github.com/project-codeflare/codeflare-sdk/blob/main/demo-notebooks/guided-demos/0_basic_ray.ipynb) assuming this is necessary  but looks this change is unnecessary the tests are pass now in local without this change,  earlier i am getting some error